### PR TITLE
17426 change factory method name

### PR DESCRIPTION
**Pull request by dillo**
## Description of change

- Renaming method in the Factory class to `create` from `create_questionnaire_response `
- Removing an unused request spec file.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Things to know about this PR

- Feature flag: show_healthcare_experience_questionnaire

- [x] RSpec tests passing

### CODE REVIEW

**Review by LindseySaari — APPROVED** [review]
👍

File: modules/health_quest/app/services/health_quest/patient_generated_data/questionnaire_response/factory.rb [hunk 1]
```diff
@@ -63,7 +63,7 @@ def search(filters)
         # @param data [Hash] questionnaire answers and appointment data hash.
         # @return [FHIR::DSTU2::Patient::ClientReply] an instance of ClientReply
         #
-        def create_questionnaire_response(data)
+        def create(data)
```
Comment (erluti) [inline modules/health_quest/app/services/health_quest/patient_generated_data/questionnaire_response/factory.rb:66]: why shorten the name?

**Review by erluti — APPROVED** [review]
LGTM, although `create` seems less descriptive than `create_questionnaire_response` and in the test that gets changed it feels less clear what's going on

File: modules/health_quest/app/services/health_quest/patient_generated_data/questionnaire_response/factory.rb (hunk 1 above)
Comment (dillo) [reply modules/health_quest/app/services/health_quest/patient_generated_data/questionnaire_response/factory.rb:66]: Just following the already established convention. We already have methods named `get` and `search` inside the class so `create_<this_thing>` seemed redundant.

**Pull request merged**